app/controllers/concerns/commentable.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Commentable
+  extend ActiveSupport::Concern
+
+  def create
+    @comment = @commentable.comments.build(comment_params)
+    @comment.user_id = current_user.id
```

Comment:
オブジェクトをそのまま代入できます。

```suggestion
    @comment.user = current_user
```

が、2 行に分けて書くのが面倒なので、自分なら以下のどちらかにします。

```ruby
@comment = @commentable.comments.build(comment_params.merge(user: current_user))
```

```ruby
@comment = current_user.comments.build(comment_params.merge(commentable: @commentable))
```

Reply:
オブジェクトを代入できたのですね！`merge`をこのような使い方ができるのは驚きでした。
修正しました。https://github.com/SuzukaHori/fjord-books_app-2023/pull/7/commits/d89f383c6839ca44b440dba4d5da935686305c02